packages/form-js-viewer/src/features/repeatRender/RepeatRenderManager.js

```diff
@@ -0,0 +1,188 @@
+// disable react hook rules as the linter is confusing the functional components within a class as class components
```

Comment:
We could also move the functional components under a `/components` sub folder and make the services available via props.

Reply:
Yeah maybe, but honestly this pattern works nicely and I trust I know what I'm doing with it :D